content/ko/docs/contribute/localization_ko.md

```diff
@@ -162,36 +163,47 @@ autoscaler | 오토스케일러 |
 availability zone | 가용성 영역(availability zone) |
 bare pod | 베어(bare) 파드 |
 beta | 베타 |
+Binding | 바인딩(Binding) |
 boilerplate | 상용구 |
 Boot | 부트 |
 Build | 빌드 |
 Cache | 캐시 |
 canary | 카나리(canary) | 릴리스 방식에 관련한 용어인 경우에 한함
 cascading | 캐스케이딩(cascading) |
+CertificateSigningRequest | CertificateSigningRequest |
 character set | 캐릭터 셋 |
 Charts | 차트 |
 checkpoint | 체크포인트 |
 CLI | CLI |
 Cluster | 클러스터 |
+ClusterRole | 클러스터롤(ClusterRole) |
+ClusterRoleBinding | 클러스터롤바인딩(ClusterRoleBinding) |
 Command Line Tool | 커맨드라인 툴 |
+ComponentStatus | 컴포넌트스테이터스(ComponentStatus) |
 Config Map | 컨피그 맵 |
+ConfigMap | 컨피그맵(ConfigMap) |
```

Comment:
Should “Config Map” go?

Reply:
I also agree with @sftim. I think it would be better to consider two terms with different capitalization or spaces as the same term and merge them into one.

Reply:
I agree.
There are several cases in the en/docs by adding a space to the API Object name. Is this the correct notation? (For examples, Config Maps, Stateful Set, Replica Set, ...)